packages/common/src/workflowHelpers.ts

```diff
@@ -310,3 +310,22 @@ export function updateWorkflowTemplateIds(
   }
   return templates;
 }
+
+/**
+ * Get the various dependencies from the workflow item
+ *
+ * @param template The IItemTemplate of the workflow item
```

Comment:
"...; the ids of found dependencies are added to the template's list of dependencies"